internal/engine/upper.go

```diff
@@ -700,3 +705,15 @@ func showError(ctx context.Context, state *store.EngineState, resourceNumber int
 }
 
 var _ model.ManifestCreator = Upper{}
+
+type newManifestErr struct {
+	s string
+}
+
+func (e *newManifestErr) Error() string { return e.s }
+
+var _ error = &newManifestErr{}
+
+func newManifestErrf(format string, a ...interface{}) *newManifestErr {
```

Comment:
I think we could remove the assertion above by changing this line to:

```suggestion
func newManifestErrf(format string, a ...interface{}) error {
```